packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -52,9 +54,9 @@ export class VaRadioOption {
         onClick={() => this.handleChange()}
         role="radio"
         value={this.value}
-        id={this.label}
+        id={this.el.id || this.label}
       >
-        <label htmlFor={this.label}>{this.label}</label>
+        <label htmlFor={this.label} aria-label={this.name}>{this.label}</label>
```

Comment:
Is this related to the `id` changes or is it a separate improvement? I'm not complaining about a11y enhancements, I'm just trying to get a better understanding.

Reply:
This is due to Courtney's comment on having an accessibility error on the radio elements. I added her screen shots to the PR in the comment above

Reply:
Issue reported by @CourtneyChambers77 resolved via these steps: https://www.ssa.gov/accessibility/andi/help/alerts.html?no_name_generic

Screenshot to verify
<img width="1396" alt="Screen Shot 2022-02-03 at 12 06 20 PM" src="https://user-images.githubusercontent.com/11822533/152392887-145afb8c-72a4-4f04-aa44-1452f763e70a.png">